### PR TITLE
Properly integrate current user handling

### PR DESCRIPTION
**Pull request by pmuir**
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**

* **Other information**:

### CODE REVIEW

File: src/app/profile/overview/overview.component.html [hunk 1]
```diff
@@ -1,40 +1,40 @@
...
+      </div>
+      <div class="row">
+        <h2>{{context.current.entity.attributes.fullName}}</h2>
+      </div>
+      <div class="row">
+        <p>
+          {{context.current.entity.attributes.username}}
+        </p>
+        <p>
+          {{context.current.entity.attributes.bio}}
```
Comment (joshuawilson) [inline src/app/profile/overview/overview.component.html]: I'm unsure but it appears that attributes is unresolved here.

File: src/app/dummy/dummy.service.ts [hunk 2]
```diff
@@ -344,6 +350,17 @@ export class DummyService {
...
+      message => {
+        this.addUser(message);
+        this._currentUser = this.lookupUser(message.attributes.username);
+      }
+    );
+    this.broadcaster.on<string>('logout').subscribe(
+      message => {
+        this._currentUser = null;
+      }
+    )
```
Comment (joshuawilson) [inline src/app/dummy/dummy.service.ts]: This statement is missing the closing semi-colon.

File: src/app/profile/overview/overview.component.html (hunk 1 above)
Comment (pmuir) [inline src/app/profile/overview/overview.component.html]: Yeah, I wasn't sure how to best do a cast here. I kinda need something like:

   {{ (context.current.entity as User).attributes.bio }}

Any ideas?

File: src/app/dummy/dummy.service.ts (hunk 2 above)
Comment (pmuir) [inline src/app/dummy/dummy.service.ts]: See #44

**Pull request merged**